### PR TITLE
Inspect: Use AutoSizer for managing width for content in tabs.

### PR DESCRIPTION
**Pull request by peterholmberg**
**What this PR does / why we need it**:
The width for the data table in the panel inspect was hard coded which will lead to issues with different screens. This PR wraps the content in an autosizer.

Also, migrating from `react-virtualized` to `react-virtualized-auto-sizer`

### CODE REVIEW

**Comment by dprokop** [comment]
@peterholmberg can we also fix https://github.com/grafana/grafana/issues/21409 in this issue?

**Comment by peterholmberg** [comment]
Yes @dprokop, I realized that when I created this PR :smile_cat:

File: packages/grafana-ui/package.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
     "@types/react-custom-scrollbars": "4.0.5",
     "@types/react-test-renderer": "16.9.0",
     "@types/react-transition-group": "2.0.16",
+    "@types/react-virtualized-auto-sizer": "1.0.0",
```
Comment (dprokop) [inline packages/grafana-ui/package.json]: Let's remove it from grafana-ui, as it's used in withFullSizeStory.tsx which is not used anywhere (let's remove it too :D)

**Review by dprokop — APPROVED** [review]
Looking good, let's juse get rid of this dependency from grafana-ui

Created #21529 as a folowup to replace react-sizeme

File: public/app/features/dashboard/components/Inspector/PanelInspector.tsx [hunk 2]
```diff
@@ -174,17 +175,28 @@ export class PanelInspector extends PureComponent<Props, State> {
...
-
-          {tab === InspectTab.Issue && this.renderIssueTab()}
-
-          {tab === InspectTab.Raw && (
-            <div>
-              <JSONFormatter json={last} open={2} />
-            </div>
-          )}
+          <AutoSizer disableHeight>
+            {({ width }) => {
```
Comment (torkelo) [inline public/app/features/dashboard/components/Inspector/PanelInspector.tsx]: dont we need height as well? table should use available height

**Pull request merged**